overrides.nix

```diff
@@ -1763,6 +1763,13 @@ self: super:
   # nix uses a dash, poetry uses an underscore
   typing_extensions = super.typing_extensions or self.typing-extensions;
 
+  typing-extensions = super.packaging.overridePythonAttrs (
```

Comment:
Shouldn't it be `super.typing-extensions.overridePythonAttrs` ?

Reply:
@lovesegfault Can you address this?